src/index.ts

```diff
@@ -11,6 +11,10 @@ export class PromisePool<T = unknown> {
     return Promise.all(this.promises.values());
   }
 
+  setConcurrency(newConcurrency: number): void {
```

Comment:
Could you add getter and setter instead of `setConcurrency`?

c.f. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get
and https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/set

Reply:
I changed the code to use getter and setter.